src/test/java/io/outfoxx/swiftpoet/test/FunctionSpecTests.kt

```diff
@@ -558,6 +558,31 @@ class FunctionSpecTests {
     )
   }
 
+  @Test
+  @DisplayName("Generates parameter modifiers")
+  fun testGenParameterDefault() {
+    val testClass = FunctionSpec.builder("test")
+      .addParameter(
+        ParameterSpec.builder("_ a", INT)
```

Comment:
You don't create them this way... Use the `ParameterSpec.builder("_", "a", INT)` version.

Reply:
Thanks! Also I'll work on splitting this up as soon as I have time. Previous changes were rather urgent for a work project.